composeApp/src/commonMain/kotlin/sp/bvantur/tasky/core/data/local/SyncStep.kt

```diff
@@ -0,0 +1,10 @@
+package sp.bvantur.tasky.core.data.local
+
+enum class SyncStep {
+    NONE,
+    CREATE,
+    EDIT,
+    DELETE;
```

Comment:
Does `NONE` mean it should not get synced? Or does it mean it is fully synced? It is not quite clear by the name.

Reply:
It's saying if sync is needed or not. In what sync state is. If it is anything else then NONE it requires syncing. Would you have some other Sync step name for elements that are fully synced?

Reply:
Yes it also reads like an IDLE state to me